galley/pkg/config/analysis/analyzers/util/config.go

```diff
@@ -32,6 +33,8 @@ var (
 		constants.KubePublicNamespace,
 		constants.KubeNodeLeaseNamespace,
 		constants.LocalPathStorageNamespace,
+
```

Comment:
this list is used for a lot of other purposes, seems like this may negatively impact those

Reply:
I missed that the list is used by _bugreport.go_.  I feel like we analysis needs a function to tell if a namespace doesn't run workloads and this is the function.  I made a slight change so it doesn't affect _bugreport.go_.  If you had something else in mind please elaborate.

Reply:
I thought it was more original, but looks like it was just bugreport